src/Form/TypeExtension/FormTypeExtension.php

```diff
@@ -17,13 +19,25 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         }
     }
 
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        $view->vars['container_attr'] = [];
+
+        if ($options['form_full']) {
+            $view->vars['container_attr']['class'] = 'form__row';
+            $view->vars['attr']['class'] = isset($view->vars['attr']['class']) ? $view->vars['attr']['class'].' form--full' : 'form--full';
```

Comment:
This stuff belong to the view logic IMO. We don't need a form type option just to add a class to a form.

Reply:
I should have this things in twig then ?

Reply:
Yes, wherever you want but not here :D